frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -289,6 +289,7 @@ class AccountsSummary extends Component<Props, State> {
   public render() {
     const { t, accounts } = this.props;
     const { exported, data, balances } = this.state;
+
```

Comment:
OCD nit 🤓  this file has no other changes, please undo the newline for nice commit history.

Reply:
done :+1: